code/_byond_version_compat.dm

```diff
@@ -0,0 +1,49 @@
+// This file contains defines allowing targeting byond versions newer than the supported
+
+//Update this whenever you need to take advantage of more recent byond features
+#define MIN_COMPILER_VERSION 514
+#define MIN_COMPILER_BUILD 1556
```

Comment:
Why add this when we have the build blacklist system, that also has min version stuff…

Reply:
This is actually DM-side version for compiler, not client-side version

Reply:
aha